README.rst

```diff
@@ -236,29 +236,30 @@ Or, to install the latest beta or release candidate, if any:
 In order to provide an extra speed boost, Falcon can compile itself with
 Cython. Wheels containing pre-compiled binaries are available from PyPI for
 several common platforms. However, if a wheel for your platform of choice is not
-available, you can choose to stick with the source distribution, or use the
-instructions below to cythonize Falcon for your environment.
-
-The following commands tell pip to install Cython, and then to invoke
-Falcon's ``setup.py``, which will in turn detect the presence of Cython
-and then compile (AKA cythonize) the Falcon framework with the system's
-default C compiler.
+available, you can install the source distribution, that will automatically
+try to cythonize Falcon for your environment, falling back to a normal
+pure-Python install in case of issues in the cythonization step:
 
 .. code:: bash
 
-    $ pip install cython
-    $ pip install --no-build-isolation --no-binary :all: falcon
-
-Note that ``--no-build-isolation`` is necessary to override pip's default
-PEP 517 behavior that can cause Cython not to be found in the build
-environment.
+    $ pip install --no-binary :all: falcon
 
 If you want to verify that Cython is being invoked, simply
-pass `-v` to pip in order to echo the compilation commands:
+pass the verbose flag `-v` to pip in order to echo the compilation commands.
+You should see an log that informs you if the cython compile was successful
+of it the fallback normal install was used:
+
+.. code:: bash
+
+    $ pip install -v --no-binary :all: falcon
+
+If for some reason you want to skip Cython compilation step and install
```

Comment:
I think it might be worth reassuring the reader that we automatically choose to skip Cython compilation on platforms where it is not beneficial performance wise, such as PyPy, and they don't need to set any variables if that is the case.